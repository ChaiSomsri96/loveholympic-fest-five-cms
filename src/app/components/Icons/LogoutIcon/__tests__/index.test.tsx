import * as React from 'react';
import { render } from '@testing-library/react';

import { LogoutIcon } from '..';

jest.mock('react-i18next', () => ({
  useTranslation: () => {
    return {
      t: str => str,
      i18n: {
        changeLanguage: () => new Promise(() => {}),
      },
    };
  },
}));

describe('<LogoutIcon  />', () => {
  it('should match snapshot', () => {
    const loadingIndicator = render(<LogoutIcon />);
    expect(loadingIndicator.container.firstChild).toMatchSnapshot();
  });
});
