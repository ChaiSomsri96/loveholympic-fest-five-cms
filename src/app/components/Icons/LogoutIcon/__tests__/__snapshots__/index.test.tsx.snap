// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<LogoutIcon  /> should match snapshot 1`] = `
<svg
  fill="none"
  height="22"
  viewBox="0 0 20 22"
  width="20"
  xmlns="http://www.w3.org/2000/svg"
>
  <path
    d="M11 1L1 13H10L9 21L19 9H10L11 1Z"
    stroke="white"
    stroke-linecap="round"
    stroke-linejoin="round"
    stroke-width="2"
  />
</svg>
`;
